### PR TITLE
Hotfix rm ordernumber

### PR DESCRIPTION
**Pull request by rmarchandTen24**
This change is [Reviewable](https://reviewable.io/reviews/ten24/slatwall/5835)

### CODE REVIEW

File: config/scripts/preupdate/v5_1.cfm [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+	<cfif !currenttables.recordCount>
+		<cfquery datasource="#this.datasource.name#" name="createSwOrderNumber">
+			CREATE TABLE SwOrderNumber(
+				orderNumber int ,
+				orderID VARCHAR(32),
+				createdDateTime TIMESTAMP
+			);
+		</cfquery>
+	</cfif>
+
```
Comment (sumitverma) [inline config/scripts/preupdate/v5_1.cfm]: Why do we need to insert all the order values? You can set the seed like this:

lastValue = select max(orderNumber) from SwOrder;
ALTER TABLE SwOrderNumber AUTO_INCREMENT = lastValue;

File: config/scripts/preupdate/v5_1.cfm [hunk 2]
```diff
@@ -0,0 +1,37 @@
+<cfparam name="this.ormSettings.dialect" />
+<cfparam name="this.datasource.name" />
+<cfparam name="this.datasource.username" default="" />
+<cfparam name="this.datasource.password" default="" />
+
+<cfsetting requesttimeout="1200" />
+<cfif ListFind(this.ormSettings.dialect, 'MySQL')>
```
Comment (sumitverma) [inline config/scripts/preupdate/v5_1.cfm]: We can do this for both mssql and mysql, mssql will be identity

File: Application.cfc [hunk 3]
```diff
@@ -47,9 +47,12 @@ Notes:
 	
 */
 component extends="org.Hibachi.Hibachi" output="false" {
-
 	// ===================================== HIBACHI HOOKS
 	
+	public void function onApplicationStart() {
+		setting requesttimeout="180";
+	}
```
Comment (sumitverma) [inline Application.cfc]: I don't think we need this because we do this in Hibachi.cfc

File: model/entity/Order.cfc [hunk 4]
```diff
@@ -269,12 +269,24 @@ component displayname="Order" entityname="SlatwallOrder" table="SwOrder" persist
...
-				if( arrayIsDefined(maxOrderNumber,1) ){
-					setOrderNumber(maxOrderNumber[1] + 1);
-				} else {
-					setOrderNumber(1);
+				if(getApplicationValue('databaseType') eq "MySQL"){
+					var maxOrderNumberSQL = 'insert into SwOrderNumber (orderID) VALUES (:orderID)';
+
+					var maxOrderNumberQuery = new query();
+					maxOrderNumberQuery.setSQL(maxOrderNumberSQL);
+					maxOrderNumberQuery.addParam(name="orderID",value=this.getOrderID());
```
Comment (sumitverma) [inline model/entity/Order.cfc]: Can we set datetime as well?

**Comment by rmarchandTen24** [comment]
@sumitverma seems to be having issues on circle I opened a new branch and am trying to debug

**Pull request closed without merging**